application/src/main/java/com/sanction/thunder/email/EmailModule.java

```diff
@@ -17,6 +17,10 @@
 
 @Module
 public class EmailModule {
+  private static final String DEFAULT_SUCCESS_PAGE = "success.html";
+  private static final String DEFAULT_VERIFICATION_HTML = "verification.html";
+  private static final String DEFAULT_VERIFICATION_TEXT = "verification.txt";
```

Comment:
Good call here. I'm not a fan of hardcoding file names but I suppose that will be address in a future PR.